README.md

```diff
@@ -50,31 +50,32 @@ module "y" {
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
-| availability_zones |  | map | `<map>` | no |
-| aws_region | The Amazon region | string | - | yes |
-| cidr_block |  | string | `10.0.0.0/16` | no |
-| create_private_hosted_zone |  | string | `true` | no |
-| create_private_subnets |  | string | `true` | no |
-| create_s3_vpc_endpoint | Whether to create a VPC Endpoint for S3, so the S3 buckets can be used from within the VPC without using the NAT gateway. | string | `true` | no |
-| environment |  | string | - | yes |
-| project |  | string | `` | no |
-| public_subnet_map_public_ip_on_launch |  | string | `false` | no |
+| availability\_zones |  | map | `<map>` | no |
+| aws\_region | The Amazon region | string | n/a | yes |
+| cidr\_block |  | string | `"10.0.0.0/16"` | no |
+| create\_private\_hosted\_zone |  | string | `"true"` | no |
+| create\_private\_subnets |  | string | `"true"` | no |
+| create\_s3\_vpc\_endpoint | Whether to create a VPC Endpoint for S3, so the S3 buckets can be used from within the VPC without using the NAT gateway. | string | `"true"` | no |
+| environment |  | string | n/a | yes |
+| project |  | string | `""` | no |
+| public\_subnet\_map\_public\_ip\_on\_launch |  | string | `"false"` | no |
 | tags | Map of tags to apply on the resources | map | `<map>` | no |
 
 ## Outputs
 
 | Name | Description |
 |------|-------------|
-| availability_zones | List of the availability zones. |
-| nat_gateway_public_ip | Public IP address of the NAT gateway. |
-| private_dns_zone_id | ID of the the private DNS zone, optional. |
-| private_domain_name | Private domain name, optional. |
-| private_subnets | List of the private subnets. |
-| private_subnets_route_table |  |
-| public_subnets | List of the public subnets. |
-| public_subnets_route_table |  |
-| vpc_cidr | VPC CDIR. |
-| vpc_id | ID of the VPC. |
+| availability\_zones | List of the availability zones. |
+| nat\_gateway\_public\_ip | Public IP address of the NAT gateway. |
+| private\_dns\_zone\_id | ID of the the private DNS zone, optional. |
+| private\_domain\_name | Private domain name, optional. |
+| private\_subnets | List of the private subnets. |
+| private\_subnets\_route\_table |  |
+| public\_subnets | List of the public subnets. |
+| public\_subnets\_route\_table |  |
+| test |  |
```

Comment:
Remove test code or name to something meaningful please

Reply:
Fixed